### PR TITLE
Add a nopit check in meltdown

### PR DESCRIPTION
**Pull request by Walterkd**
Since there is no nopti check in meltdown, we add a nopti check in meltdown.

- Verification run: http://10.67.132.101/tests/4#

### CODE REVIEW

**Comment by Walterkd** [comment]
@wjn740 Please help me review it.

**Comment by wjn740** [comment]
@Walterkd you need to do a 'make tidy'.

File: tests/cpu_bugs/meltdown.pm [hunk 1]
```diff
@@ -77,12 +78,22 @@ sub check_cpu_flags {
...
 }
-
 sub run {
+    my ($self) = shift;
     select_console 'root-console';
     my $obj = meltdown->new($mitigations_list);
     #run base function testing
     $obj->do_test();
+    #extra testing for nopti
+    add_grub_cmdline_settings("nopti");
```
Comment (asmorodskyi) [inline tests/cpu_bugs/meltdown.pm]: taking into account #8999  I would say that it is plus one reason to extract this into a function :)

Comment (Walterkd) [reply tests/cpu_bugs/meltdown.pm]: Thanks, already changed

File: tests/cpu_bugs/meltdown.pm [hunk 2]
```diff
@@ -53,6 +53,7 @@ my $mitigations_list =
     },
   };
 
+
```
Comment (wjn740) [inline tests/cpu_bugs/meltdown.pm]: remove blank line

Comment (Walterkd) [reply tests/cpu_bugs/meltdown.pm]: Thanks, already changed

**Pull request merged**